qml/components-v-play/Theme.qml

```diff
@@ -46,9 +52,7 @@ QtObject {
         case "about":
             icon = "info";
             break;
-        case "locationarrow":
```

Comment:
This diff is unnecessary.  Leave `locationarrow` there and just put `map` underneath it.